mondey_backend/src/mondey_backend/models/calendarevents.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import datetime
+
+from pydantic import field_serializer
+from pydantic import field_validator
+from sqlmodel import Field
+from sqlmodel import SQLModel
+
+
+class CalendarEventBase(SQLModel):
+    title: str = ""
+    description: str = ""
+    external_link: str = ""
+    event_date: datetime.date
+
+    @field_validator("event_date", mode="before")
```

Comment:
if in the frontend we have a datetime instead of a custom string then I guess we could just use it without needing the custom validation?